### DISCUSSION TITLE
When will the next GitHub release occur?

### DISCUSSION THREAD

**Discussion by wshands** [GreenleafLab/ArchR#1259] [General]
We are using branch named 'release_1.0.2' but we need to be using a published release instead of a branch. The latest official published GitHub release is release_1.0.1. When will a published release from branch 'release_1.0.2' happen?

**Comment by rcorces** [comment]
We're working on it. The primary developer (Jeff) left the project last year so it has been challenging to keep everything going in his absence. We're hoping to push a stable release_1.02 in the next few months that includes an update to documentation and a new way to control all of the complex dependencies in ArchR which have caused problems for users. Thanks for your patience.

**Reply by wshands** [reply]
We really need a tag for the release_1.0.2 branch we are using. Could we have a release soon?

**Reply by rcorces** [reply]
Still working on it. No ETA

**Reply by rcorces** [reply]
In case this is helpful to you, you can always reference the specific commit SHA which can be used to install the specific codebase of ArchR you are using via `devtools::install_github()`

**Reply by wshands** [reply]
Thanks that worked
